src/components/toggle/toggle.scss

```diff
@@ -0,0 +1,42 @@
+@import "elevation";
+@import "default-theme";
+@import "palette";
+@import "mixins";
+
+$md-padding: 0 16px !default;
+$md-line-height: 36px !default;
+$md-border-radius: 3px !default;
```

Comment:
These variables should have `toggle` (or `toggle-button` if we change it) in their name, e.g., `$md-toggle-button-padding`.